service_storagefacility/app/controllers/StorageController.scala

```diff
@@ -484,8 +484,8 @@ final class StorageController @Inject()(
   def search(
       mid: Int,
       searchStr: Option[String],
-      page: Int = 1,
```

Comment:
I didnt see any changes to the routes file for this code. So then i assume the routes file had default values then ? I did in fact not take the time to look at Either the branch or master to see if route file had defaults. So the review was purely based on the diff in the PR.